sample_pages/esp/eids-esp.1.0.js

```diff
@@ -0,0 +1,39 @@
+function fetchAsyncSignals(mode, customFunction, customKey) {
```

Comment:
![Codacy](https://codacy.pubmatic.com:443/assets/images/favicon.png) Issue found: ['fetchAsyncSignals' is defined but never used.](https://codacy.pubmatic.com:443/manual/dattaprasad.mundada/OpenWrap/pullRequest?prid=15858)